vscode/package.json

```
@@ -31,6 +31,17 @@
   "activationEvents": [
     "onLanguage:json"
   ],
-  "contributes": {
+  "contributes": {                       
+    "configuration": {
+      "title": "Configuration for JSON Schema Language Server",
+        "properties": {
+          "jsonSchemaLanguageServer.defaultDialect": {
+            "type": "string",
+            "description": "The default JSON Schema dialect to use if none is specified in the schema document",
+            "default": "http://json-schema.org/draft-07/schema#"
```

There should be no default.

```suggestion
            "description": "The default JSON Schema dialect to use if none is specified in the schema document"
```